Networking/NetworkingTests/Remote/SitePluginsRemoteTests.swift

```diff
@@ -64,4 +64,142 @@ class SitePluginsRemoteTests: XCTestCase {
             XCTAssertNotNil(error)
         }
     }
+
+    // MARK: - Install plugin tests
+
+    /// Verifies that installPlugin properly parses the sample response.
+    ///
+    func test_installPlugin_properly_returns_plugins() {
+        let remote = SitePluginsRemote(network: network)
+
+        network.simulateResponse(requestUrlSuffix: "plugins", filename: "plugin")
+
+        // When
+        let result: Result<SitePlugin, Error> = waitFor { promise in
+            remote.installPlugin(for: self.sampleSiteID, slug: "jetpack") { result in
+                promise(result)
+            }
+        }
+
+        // Then
+        switch result {
+        case .success(let plugin):
+            XCTAssertEqual(plugin.plugin, "jetpack/jetpack")
+        case .failure(let error):
+            XCTAssertNil(error)
+        }
```

Comment:
nit: this switch case looks like the result could be either case, though I think we expect the result to be always successful? maybe something like this would look more clear:

```suggestion
        let plugin = try XCTUnwrap(result.get())
        XCTAssertEqual(plugin.plugin, "jetpack/jetpack")
```

(and marking the function as failable with `throws`)